bluesky_widgets/qt/search_input.py

```diff
@@ -113,14 +113,14 @@ def time_validator(since=None, until=None):
         self.days_widget.clicked.connect(self.on_select_30days)
         self.all_widget.clicked.connect(self.on_select_all)
 
-    def on_reload(self, event):
+    def on_reload(self):
```

Comment:
What input argument is needed here?

Reply:
The model will call it when a generic `Event` (no `date` in this case, but some general info about where it came from). I think it's passed as a positional argument so `on_reload(self, literally_anything)` should work  but `on_reload(self, event)` would be most readable.

So --- we have to accept `event` as an argument because of how signaling works, but in this case we aren't going to _do_ anything with it.

Reply:
Ok... I will accept this :)